ViewModel/Alternatives/Results/DamageWithUncertaintyVM.cs

```diff
@@ -29,12 +29,12 @@ public DamageWithUncertaintyVM(AlternativeResults results, DamageMeasureYear dam
             if (double.IsNaN(discountRate))
             {
                 RateAndPeriodVisible = false;
-                ProbabilityExceedsValueLabel = "Probability that damage exceeds indicated value";
```

Comment:
For labels like these you were using a String Constants class for awhile. I've also seen in other people's code having string constants at the top of the class, just so they're in the same class they're used, but still easy to find and change easily. Does it make sense to use one of those strategies here?

Reply:
There's a few places you're using string labels in code in this pull request. I'll let this comment refer to  all of them.

Reply:
I agree, i will add them to the string constants class.